clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -110,19 +114,15 @@ void visualstudio::Linker::ConstructJob(Compilation &C, const JobAction &JA,
                             llvm::archToLegacyVCArch(TC.getArch()));
     CmdArgs.push_back(Args.MakeArgString(Twine("-libpath:") + DIAPath));
   }
-  if (!llvm::sys::Process::GetEnv("LIB") ||
-      Args.getLastArg(options::OPT__SLASH_vctoolsdir,
-                      options::OPT__SLASH_winsysroot)) {
+  if (!llvm::sys::Process::GetEnv("LIB") || A) {
```

Comment:
Hmm, is the goal here to prefer `LIB` over the arguments?

Reply:
I changed this to narrow down the scope.